spec/rails_helper.rb

```diff
@@ -4,6 +4,8 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
+require 'devise'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.